svir/dialogs/load_gmf_data_as_layer_dialog.py

```diff
@@ -64,8 +64,39 @@ def __init__(self, iface, viewer_dock, session, hostname, calc_id,
     def set_ok_button(self):
         self.ok_button.setEnabled(self.imt_cbx.currentIndex() != -1)
 
-    def on_rlz_or_stat_changed(self):
-        self.dataset = self.npz_file[self.rlz_or_stat_cbx.currentText()]
+    def populate_out_dep_widgets(self):
+        self.populate_gmpe_cbx()
+        self.show_num_sites()
+
+    def _format_rlz(self, gmpe_idx):
+        # NOTE: assuming that rlzs are always in the format rlz-XXX
+        return "rlz-%03d" % gmpe_idx
+
+    def show_num_sites(self):
+        # NOTE: we are assuming all realizations have the same number of sites,
+        #       which currently is always true.
+        #       If different realizations have a different number of sites, we
```

Comment:
which currently is always true -> which is always true for scenario calculations